cmd/hz/generator/client.go

```diff
@@ -43,7 +43,8 @@ type ClientFile struct {
 
 func (pkgGen *HttpPackageGenerator) genClient(pkg *HttpPackage, clientDir string) error {
 	for _, s := range pkg.Services {
-		hertzClientPath := filepath.Join(clientDir, "hertz_client.go")
+		cliDir := util.SubDir(clientDir, util.ToSnakeCase(s.Name))
+		hertzClientPath := filepath.Join(cliDir, "hertz_client.go")
```

Comment:
Add a const for `hertz_client.go`?

Reply:
done~, in [pr](https://github.com/cloudwego/hertz/pull/511)